cuckoo/constants.py

```diff
@@ -22,17 +30,24 @@
 
 HASURA_PORT = int(os.environ.get("HASURA_PORT", "8080"))
 HASURA_HEALTH_URL = os.environ.get("HASURA_HEALTH_URL", "")
-HASURA_URL = os.environ["HASURA_URL"]
-HASURA_ADMIN_SECRET = os.environ["HASURA_ADMIN_SECRET"]
-HASURA_ROLE = os.environ["HASURA_ROLE"]
+HASURA_URL = os.environ.get("HASURA_URL")
+HASURA_ADMIN_SECRET = os.environ.get("HASURA_ADMIN_SECRET")
+HASURA_ROLE = os.environ.get("HASURA_ROLE")
```

Comment:
since you can set these through the `Cuckoo.configure()` method, they can be optional now

Reply:
You read my mind, we will literally use this immediately in the new sparse-tool! 

See: https://github.com/DroneBase/duck-sparse-tool/blob/fa25ec1d3b5568c3eab7822913969e02647c9070/duck_sparse_tool/constants.py#L29